### PR TITLE
Bugfix/tp2squadmate locality issues

### PR DESCRIPTION
**Pull request by Crowdedlight**
Fixes the issues with showing dialog for local GM and teleporting other players that require server execution. Moved to Rpc through the ``SCR_EditorManagerEntity``.

Also renamed the listbox to better reflect the type of listbox it is.

### CODE REVIEW

File: Scripts/Game/ODIN/Editor/Containers/Actions/ContextActions/ODIN_TeleportToSquadmateContextAction.c [hunk 1]
```diff
@@ -105,35 +105,19 @@ class ODIN_TeleportToSquadmateContextAction: SCR_BaseContextAction
...
-		
-		hoveredEntity.SetTransform(target_transform, true);
+		// get target player RplId
+		RplComponent targetRpl = RplComponent.Cast(selectedPlayer.FindComponent(RplComponent));
+		if (!targetRpl)
+			return;
+					
+		// Call RPC through EditorManagerEntity to reach server 
+		SCR_EditorManagerEntity managerEntity = SCR_EditorManagerEntity.GetInstance();
+		managerEntity.ODIN_TeleportEntityToPlayer(hoveredEntity.GetOwnerRplId(), targetRpl.Id(), emptyTerrainAreaRadius);
```
Comment (Kexanone) [inline Scripts/Game/ODIN/Editor/Containers/Actions/ContextActions/ODIN_TeleportToSquadmateContextAction.c:116]: `emptyTerrainAreaRadius` is a constant, hence should be moved to `SCR_EditorManagerEntity`

File: Scripts/Game/ODIN/Editor/Entities/SCR_EditorManagerEntity.c [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+		sourceEntity.GetTransform(target_transform);
+		
+		// use new position, but keep rotation 
+		target_transform[3] = target_pos;
+		
+		// call transform
+		ODIN_SetTransform(sourceEntity, target_transform);
+	}
+	
+	void ODIN_SetTransform(IEntity sourceEntity, vector transform[4])
```
Comment (Kexanone) [inline Scripts/Game/ODIN/Editor/Entities/SCR_EditorManagerEntity.c:74]: What's the point of this function? Why not just call `SCR_EditableEntityComponent.SetTransform` directly?

**Review by Kexanone — CHANGES_REQUESTED** [review]

**Comment by Crowdedlight** [comment]
Addressed Review comments. @Kexanone

File: Scripts/Game/ODIN/Editor/Entities/SCR_EditorManagerEntity.c [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+- Editor can have a limitied state, influenced by its modes and checked by IsLimited() function:
+ + Editor is **limited** if all its modes are.
+ + If at least one mode is unlimited, the editor is **unlimited** as well.
+ + The state influences behavior of certain components, e.g., photo mode camera movement may be restricted if the editor is limited.
+ + As a rule of thumb, unlimited state means that player can ***cheat*** with the editor.
+- Default editor manager prefab is defined in SCR_EditorManagerCore (config is Configs/Core folder).
+*/
+modded class SCR_EditorManagerEntity : SCR_EditorBaseEntity
+{
+	static int emptyTerrainAreaRadius = 20;
```
Comment (Kexanone) [inline Scripts/Game/ODIN/Editor/Entities/SCR_EditorManagerEntity.c:23]: Should be prefixed

**Review by Kexanone — CHANGES_REQUESTED** [review]

**Review by Kexanone — APPROVED** [review]

**Pull request merged**